wcfsetup/install/files/ts/WoltLabSuite/Core/Ajax/Data.ts

```diff
@@ -28,6 +28,7 @@ export interface DatabaseObjectActionResponse extends ResponseData {
     | any[];
 }
 
+// Return `false` to suppress the error message.
```

Comment:
```suggestion
/** Return `false` to suppress the error message. */
```
Should help the IDE picking up the comment, because `/**` usually indicates doc comments.